lib/generators/USAGE

```diff
@@ -0,0 +1,125 @@
+generator runs will get you up and running efficiently. There's a fair bit of
+setup with integrating Webpack with Rails. Defaults for options are such that
+the default is for the flag to be off. For example, the default for -R is that
+redux is off, and the default of -b means that skip-bootstrap is off.
+
+* Redux
+
+    If you have used the --redux generator option, you will notice the familiar
+    additional redux folders. The Hello World example has also been modified to
+    use Redux.
+
+    Note the organizational paradigm of "bundles". These are like application
+    domains and are used for grouping your code into webpack bundles, in case
+    you decide to create different bundles for deployment. This is also useful
+    for separating out logical parts of your application. The concept is that
+    each bundle will have it's own Redux store. If you have code that you want
+    to reuse across bundles, including components and reducers, place them
+    under /client/app/lib.
+
+* Using Images and Fonts
+
+    The generator has amended the folders created in `client/assets/` to Rails's
+    asset path. We recommend that if you have any existing assets that you want
+    to use with your client code, you should move them to these folders and use
+    webpack as normal. This allows webpack's development server to have access
+    to your assets, as it will not be able to see any assets in the default Rails
+    directories which are above the `/client` directory.
+
+    Alternatively, if you have many existing assets and don't wish to move them,
+    you could consider creating symlinks from `client/assets` that point to your
+    Rails assets folders inside of `app/assets/`. The assets there will then be
+    visible to both Rails and webpack.
+
+* Bootstrap Integration
+
+    React on Rails ships with Twitter Bootstrap already integrated into the build.
+    Note that the generator removes require_tree in both the application.js and
+    application.css.scss files. This is to ensure the correct load order for the
+    bootstrap integration, and is usually a good idea in general. You will therefore
+    need to explicitly require your files.
+
+    How the Bootstrap library is loaded depends upon whether one is using the Rails
+    server or the HMR development server.
+
+    1. Bootstrap via Rails Server
+
+        In the former case, the Rails server loads bootstrap-sprockets, provided
+        by the bootstrap-sass ruby gem (added automatically to your Gemfile by
+        the generator) via the `app/assets/stylesheets/_bootstrap-custom.scss`
+        partial.
+
+        This allows for using Bootstrap in your regular Rails stylesheets. If you
+        wish to customize any of the Bootstrap variables, you can do so via the
+        `client/assets/stylesheets/_pre-bootstrap.scss` partial.
+
+    2. Bootstrap via Webpack Dev Server
+
+        When using the webpack dev server, which does not go through Rails,
```

Comment:
Try to avoid passive voice. Replace 61 thorugh 63 with:

"The webpack dev server does not go through Rails but instead loads bootstrap via the `bootstrap-sass-loader` webpack loader. You can configure the loader via the `client/bootstrap-sass-config.js` file."